### PR TITLE
Ensure Redis 4 is used when specified

### PR DESCRIPTION
**Pull request by danielbachhuber**
From #214

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ before_script:
         phpcs --config-set installed_paths $HOME/.config/composer/vendor/wp-coding-standards/wpcs
       fi
     - |
-      if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
+      if [[ "$REDIS_VERSION" == "4" ]] ; then
```
Comment (szepeviktor) [inline .travis.yml]: What a trick!!

**Comment by szepeviktor** [comment]
@danielbachhuber Seem like it is "noop"

> Unpacking redis-server (4:4.0.8-1chl1~trusty1) over (4:4.0.6-1chl1~trusty1) ...

**Comment by danielbachhuber** [comment]
@szepeviktor Huh. So Travis comes with Redis 4 by default then?

**Comment by szepeviktor** [comment]
It seems like it does.

**Comment by szepeviktor** [comment]
Then we miss Redis v3 testing...

**Pull request closed without merging**